setup.py

```diff
@@ -21,6 +21,9 @@
         "Twisted[tls]>=14.0",
         "pyOpenSSL>=0.14",
     ],
+    extras_require = {
+        "dehydrated": ["pem"]
```

Comment:
Would you mind *either*

1. submitting a separate PR first to get rid of https://github.com/glyph/txsni/blob/5014c141a7acef63e20fcf6c36fa07f0cd754ce1/txsni/only_noticed_pypi_pem_after_i_wrote_this.py

*or*

2. just using the existing thing that txsni is using?

The existing code is a little gross, but I'd rather do this one way or the other; either a hard dependency on `pem` regardless, or using the same internal utility function?